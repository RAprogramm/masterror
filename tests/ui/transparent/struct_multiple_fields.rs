use masterror::Error;

#[derive(Debug, Error)]
#[error(transparent)]
struct TransparentMany {
    first: String,
    second: String
}
