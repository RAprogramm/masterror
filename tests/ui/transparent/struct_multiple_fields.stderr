error: using #[error(transparent)] in struct `TransparentMany` requires exactly one field
 --> tests/ui/transparent/struct_multiple_fields.rs:4:1
  |
4 | #[error(transparent)]
  | ^

error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/ui/transparent/struct_multiple_fields.rs:8:2
  |
8 | }
  |  ^ consider adding a `main` function to `$DIR/tests/ui/transparent/struct_multiple_fields.rs`
